code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -783,3 +783,60 @@
 	var/location = get_turf(holder.my_atom)
 	for(var/i in 1 to created_volume)
 		new /obj/item/stack/sheet/ashresin(location)
+
+/datum/chemical_reaction/sulfur_trioxide
+	name = /datum/reagent/sulfur_trioxide
+	id = /datum/reagent/sulfur_trioxide
+	results = list(/datum/reagent/sulfur_trioxide = 2)
+	required_reagents = list(/datum/reagent/sulfur = 1, /datum/reagent/oxygen = 3)
+	required_catalysts = list(/datum/reagent/hydrogen = 1)
+
+/datum/chemical_reaction/sulfuric_acid
+	name = /datum/reagent/toxin/acid
+	id = /datum/reagent/toxin/acid
+	results = list(/datum/reagent/toxin/acid = 1)
+	required_reagents = list(/datum/reagent/sulfur_trioxide = 1, /datum/reagent/water = 1)
+
+/datum/chemical_reaction/sugar
+	name = /datum/reagent/consumable/sugar
+	id = /datum/reagent/consumable/sugar
+	results = list(/datum/reagent/consumable/sugar = 1)
+	required_reagents = list(/datum/reagent/carbon = 1, /datum/reagent/oxygen = 1, /datum/reagent/fuel = 1)
+
+/datum/chemical_reaction/welding_fuel
+	name = /datum/reagent/fuel
+	id = /datum/reagent/fuel
+	results = list(/datum/reagent/fuel = 1)
+	required_reagents = list(/datum/reagent/oil = 1, /datum/reagent/toxin/acid = 3)
```

Comment:
this creates an ouroboros wherein you can use oil to make welding fuel to make oil to make welding fuel (except it's net negative)

This has no bearing on anything I just felt like pointing it out

Reply:
lmao, right, forgot oil uses welding fuel for....Some Reason